config/Dockerfiles/tests.d/dummy/03_dummy-creds-vault

```diff
@@ -0,0 +1,34 @@
+#!/bin/bash -xe
+
+# Verify creds-vault on dummy provider
+# distros.exclude: none
+# providers.include: dummy
+# providers.exclude: none
+
+DISTRO=${1}
+PROVIDER=${2}
+
+PINFILE="PinFile"
+TARGET="dummy_target"
+WORKSPACE_PATH="docs/source/examples/workspaces/dummy-creds-vault/"
+CREDS_PATH="docs/source/examples/workspaces/dummy-creds-vault/credentials/"
+export VAULT_PASSWORD=testval
```

Comment:
The dummy provider doesn't use credentials, which seems like this test would not be useful. Additionally, the default location for testing credentials are actually stored in Jenkins variables. This credential itself should probably be stored in the contra-files repository, and included in some way where it can be expanded, rather than specifying the location here.

Reply:
@herlo : Dummy provider does not use credentials but the above changes simulates it use of credentials when mentioned in topology . The purpose of the current PR is to check whether the ansible vault is working or not . Further , The dummy credentials used are mock values I would not want them to mix up with the working credentials That is one of the reasons i chose to keep it within the workspace.

Reply:
Ok, thanks for the explanation.